src/site/views/event/tmpl/default.php

```diff
@@ -117,7 +117,10 @@
 							foreach ($eventAttendees as $person)
 							{
 								$universityCounts[$person['Uni']] += 1;
-								$levelCounts[$person['Level']] += 1;
+								if (strpos($person['Ticket'], 'Party') === false && strpos($person['Ticket'], 'party') === false)
```

Comment:
Ergh! I'm not a massive fan of checking to see if the ticket name contains a string. What if it's accidentally misspelled or it's called a different name (e.g. "Partay")? How about not providing this table and if a uni wants these numbers then can put the all attendees table in a spreadsheet and work it out per event?

Reply:
yeah i know, part of me realllly hates this but the only reason im doing any of these tables is i get asked for them every event, and have to create a google sheet of them every time

Reply:
A cleaner way of doing this would be add three "level" columns to the top "ticket sales" table, but that would require a larger amount of sql joining and stuff in the model, and an unweildy table

Reply:
Cool, well I'll approve this then and we can always redo it later if we so desire.